404.html

```diff
@@ -35,4 +181,56 @@ <h2 class="tagline push-bottom">We're sorry. This page doesn't seem to exist.</h
+<p><a href="https://www.crossroads.net/live/" class="ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken ss-broken">Anywhere</a><br><a href="https://www.crossroads.net/cleveland/">Cleveland</a><br><a href="https://www.crossroads.net/columbus/">Columbus</a><br><a href="https://www.crossroads.net/dayton/">Dayton</a></p>
+</div>
+<div class="col-sm-2 push-bottom">
+<h5><a href="http://imincampaign.net/" target="_blank">I’M IN Campaign</a></h5>
+<p><!--<a href="[sitetree_link,id=2876]">Commit to I'M IN</a><br />--> <a href="mailto:campaign@crossroads.net">I'M IN questions</a><br><a href="https://www.crossroads.net/leaveyourmark">Campaign Status</a></p>
+</div>
+<div class="col-sm-2 open push-bottom">
+<h5>Contact</h5>
+<p><a href="https://www.crossroads.net/prayer/">Prayer Requests</a> <br><a href="https://www.crossroads.net/shareyourstory/">Share Your Story</a> <br><a href="https://www.crossroads.net/contactus/">Contact Us</a> <br><a href="https://www.crossroads.net/privacypolicy/">Privacy Policy</a></p>
+</div>
+<div class="col-sm-2 open push-bottom">
+<h5>Resources</h5>
+<p><a href="https://www.crossroads.net/inthenews/">In The News</a><br><a href="https://www.crossroads.net/churchresources/">Church Resources</a></p>
+</div>
+<div class="col-sm-4 social-icons"><a href="https://twitter.com/crdschurch/" target="_blank" class="text-center"><img src="//d1tmclqz61gqwd.cloudfront.net/images/twitter.svg" alt="twitter account" title="twitter account"></a> <a href="https://www.facebook.com/crdschurch/?_rdr=p" target="_blank" class="text-center"> <img src="//d1tmclqz61gqwd.cloudfront.net/images/facebook.svg" alt="facebook account" title="facebook account"></a> <a href="https://www.youtube.com/user/crdschurch" target="_blank" class="text-center"> <img src="//d1tmclqz61gqwd.cloudfront.net/images/youtube.svg" alt="youtube account" title="youtube account"></a> <a href="https://www.instagram.com/crdschurch/" target="_blank" class="text-center"> <img src="//d1tmclqz61gqwd.cloudfront.net/images/instagram.svg" alt="instagram account" title="instagram account"></a></div>
+</div>
+</div>
+</div>
+</div>
+
+  <!-- send data to Monetate -->
+
+  <script>
+    var CRDS = {
+      media: {
+        cms: 'https://contentint.crossroads.net/',
+        app: 'https://int.crossroads.net/',
+        img: 'https://int.crossroads.net/proxy/gateway/api/image/profile/',
+        prefix: 'int'
+      },
+      streamspotId: 'crossr30e3',
+      streamspotKey: 'a0cb38cb-8146-47c2-b11f-6d93f4647389',
+      streamspotPlayerId: '',
+      env: {
+        "gatewayServerEndpoint": 'https://gatewayint.crossroads.net/gateway/'
+      }
+    };
+  </script>
+  <script type="text/javascript" src="/assets/application-14d4a79cc6a1ae82db11930c.js" ></script>
```

Comment:
Don't think we want these preprocessed assets here. We should be able to use the frontmatter as long as we throw that permalink up there.